Library/Homebrew/cask/installer.rb

```diff
@@ -571,6 +574,106 @@ def purge_caskroom_path
       gain_permissions_remove(@cask.caskroom_path)
     end
 
+    sig { void }
+    def forbidden_tap_check
+      forbidden_taps = Homebrew::EnvConfig.forbidden_taps
+      return if forbidden_taps.blank?
+
+      forbidden_taps_set = Set.new(forbidden_taps.split.filter_map do |tap|
+        Tap.fetch(tap)
+      rescue Tap::InvalidNameError
+        opoo "Invalid tap name in `HOMEBREW_FORBIDDEN_TAPS`: #{tap}"
+      end)
```

Comment:
```suggestion
      rescue Tap::InvalidNameError
        opoo "Invalid tap name in `HOMEBREW_FORBIDDEN_TAPS`: #{tap}"
        nil
      end)
```

Because `[1].filter_map { |_| opoo "a" } == [T::Private::Types::Void::VOID]`